### ISSUE TITLE
Is there any demo for scrolling from top to refresh by App.js ? Thanks  :-)

### ISSUE THREAD

**Issue by jackyzhong123** [kikinteractive/app#76]

**Comment by jairajs89** [comment]
No. It is possible but we've never wanted it for any of our apps and nobody has asked for it before

**Comment by kohaistyle** [comment]
Not tested, but you can have a go with this : https://github.com/ccoenraets/pull-to-refresh

It seems to work ok, but i didn't test with app.js yet

**Comment by jackread** [comment]
@jackyzhong123 I've implemented it before using this https://github.com/apeatling/web-pull-to-refresh/
